prowler/__main__.py

```diff
@@ -248,16 +248,22 @@ def prowler():
     stats = extract_findings_statistics(findings)
 
     if args.slack:
-        if "SLACK_API_TOKEN" in environ and "SLACK_CHANNEL_ID" in environ:
+        if "SLACK_API_TOKEN" in environ and (
+            "SLACK_CHANNEL_NAME" in environ or "SLACK_CHANNEL_ID" in environ
+        ):
             _ = send_slack_message(
```

Comment:
Are both values valid in the Slack API?

Reply:
What API? Those values are parsed by Prowler and then used them as args to the `send_slack_message` function.

Reply:
I mean that you are now passing either `SLACK_CHANNEL_NAME` or `SLACK_CHANNEL_ID` to the `send_slack_message` function, and then is used in the following way:
```python
response = client.chat_postMessage(
            username="Prowler",
            icon_url=square_logo_img,
            channel=f"#{channel}",
            blocks=create_message_blocks(identity, logo, stats),
        )
```
Are both valid when used in the Slack API as `channel`?